tests/components/reolink/test_init.py

```diff
@@ -0,0 +1,151 @@
+"""Test the Reolink init."""
+from typing import Any
+from unittest.mock import AsyncMock, MagicMock, Mock
+
+import pytest
+from reolink_aio.exceptions import ReolinkError
+
+from homeassistant.components.reolink import const
+from homeassistant.components.reolink.config_flow import DEFAULT_PROTOCOL
+from homeassistant.config import async_process_ha_core_config
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_PORT, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import issue_registry as ir
+from homeassistant.helpers.device_registry import format_mac
+
+from .conftest import (
+    TEST_HOST,
+    TEST_MAC,
+    TEST_NVR_NAME,
+    TEST_PASSWORD,
+    TEST_PORT,
+    TEST_USE_HTTPS,
+    TEST_USERNAME,
+)
+
+from tests.common import MockConfigEntry
+
+pytestmark = pytest.mark.usefixtures("reolink_connect", "reolink_init")
+
+
+@pytest.fixture(name="config_entry")
+def reolink_config_entry_fixture(hass: HomeAssistant) -> MockConfigEntry:
```

Comment:
I suggest you move this fixture to `conftest.py`.
You will likely need it when you add platform tests later.

Reply:
Thanks for the suggestion, moved.